{"uuid": "26bca0b1-3709-45c8-b2e3-a05a90855411", "children": ["1826371e-4782-42f6-b133-278256a582ea", "d93bade8-ad48-4293-9451-b9b56dc828c0", "63c55e31-2e15-4b56-82a7-1c13e0c0974e", "c8d26974-de0d-4623-9eba-15dee973335f", "89ea4530-24ac-4086-b4e7-84b4f3dc3e3c"], "befores": [{"name": "_Class__pytest_setup_class", "status": "passed", "start": 1586835901028, "stop": 1586835901028}], "afters": [{"name": "_Class__pytest_setup_class::0", "status": "broken", "statusDetails": {"message": "NameError: name 'mail_pass' is not defined\n", "trace": "  File \"/usr/local/lib/python3.7/site-packages/allure_commons/_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/fixtures.py\", line 800, in _teardown_yield_fixture\n    next(it)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/python.py\", line 707, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/python.py\", line 634, in _call_with_optional_argument\n    func(arg)\n  File \"/Users/sonic/Project/Python/Pytest/scripts/test_case.py\", line 41, in teardown_class\n    EmailHandler().send_email()\n  File \"/Users/sonic/Project/Python/Pytest/uti/EmailHandler.py\", line 59, in send_email\n    smtp_obj.login(mail_user, mail_pass)\n"}, "start": 1586835901186, "stop": 1586835909414}], "start": 1586835901028, "stop": 1586835909414}